src/test/resources/application-test.yml

```diff
@@ -101,7 +101,7 @@ spring:
 cancel:
   queue:
     purge-frequency-time: 500ms
-    purge-total-time: 1s
+    purge-total-time: 2s
 
```

Comment:
1 second seems to short for allocation migration test - so up to 2 (was 5 originally)